satellite-placement/requirements_tests.txt

```diff
@@ -0,0 +1,2 @@
+dwave-ocean-sdk==1.2.0
+networkx==2.2
```

Comment:
Could we temporarily add `coverage` to this requirements file? In the near future, I will remove coverage from the demo-specific requirements file and make it a default requirement for all demos.

This would just be a temporary measure to keep CircleCI running.

Reply:
done